code/lib/cli/src/sandbox-templates.ts

```diff
@@ -358,7 +358,7 @@ const baseTemplates = {
   'lit-vite/default-ts': {
     name: 'Lit Latest (Vite | TypeScript)',
     script:
-      'npm create vite@latest --yes {{beforeDir}} -- --template lit-ts && cd {{beforeDir}} && echo "export default {}" > vite.config.js',
+      'npm create vite@beta --yes {{beforeDir}} -- --template lit-ts && cd {{beforeDir}} && echo "export default {}" > vite.config.js',
```

Comment:
This is temporary right, do we have a plan in place of when it's changed back to "latest"?

Reply:
Yes, temporary until Vite 5 stable is released.  Typically I go back in and submit a PR that changes it back after that happens.